main.py

```diff
@@ -46,16 +46,43 @@ def is_out_of_bounds(self):
 class ScoreCard:
     def __init__(self):
         self.position = [3, 3]
+        self.highscorepos = [3,20]
         self.score = 0
         self.font = pygame.font.Font('freesansbold.ttf', 20)
-
+        self.load_highscore()
+
+    def __del__(self): self.save_highscore()
+    
+    def load_highscore(self):
+        #load high score
+        try:
+            with open('score/highscore.txt', 'r+') as f:
+                self.highscore = int(f.read())
+        except:
+            with open('score/highscore.txt', 'w') as f:
+                self.highscore = 0
+
+    def save_highscore(self):
+        with open('score/highscore.txt', 'w') as f:
```

Comment:
Use `os.path.join` instead of directly writing file path, as it will affect interoperability across different types of system.

Reply:
Just pushed in a new update regarding these change requests!